### PR TITLE
Revert "Added in gitignore"

### PR DESCRIPTION
**Pull request by briangonzalez**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/briangonzalez/termrc/pull/18" target="_blank">ReviewNinja

File: .gitignore [hunk 1]
```diff
@@ -1,4 +0,0 @@
-.DS_store
-
-tmp/
```
Comment (briangonzalez) [inline .gitignore]: !fix Brian please remove this

Comment (briangonzalez) [inline .gitignore]: !resolved how does it look now?

**Comment by briangonzalez** [comment]
!star

**Comment by briangonzalez** [comment]
!unstar

**Comment by briangonzalez** [comment]
:+1:

**Comment by briangonzalez** [comment]
:-1:

**Pull request closed without merging**